lib/ansible/modules/monitoring/grafana_plugin.py

```diff
@@ -29,28 +30,31 @@
   version:
     description:
       - Version of the plugin to install.
-      - Default to latest.
+      - Defaults to C(latest).
   grafana_plugins_dir:
     description:
-      - Directory where Grafana plugin will be installed.
+      - Directory where the Grafana plugin will be installed.
+      - If omitted, defaults to C(/var/lib/grafana/plugins).
   grafana_repo:
     description:
-      - Grafana repository. If not set, gafana-cli will use the default value C(https://grafana.net/api/plugins).
+      - URL to the Grafana plugin repository.
+      - "If omitted, grafana-cli will use the default value: L(https://grafana.com/api/plugins)."
   grafana_plugin_url:
     description:
-      - Custom Grafana plugin URL.
+      - Full URL to the plugin zip file instead of downloading the file from L(https://grafana.com/api/plugins).
       - Requires grafana 4.6.x or later.
   state:
     description:
-      - Status of the Grafana plugin.
-      - If latest is set, the version parameter will be ignored.
```

Comment:
What's your thinking about removing this note?

Reply:
`latest` isn't a choice for state, so the statement is incorrect.